### ISSUE TITLE
bugs in exp_imputation.py

### ISSUE THREAD

**Issue by liyiersan** [thuml/Time-Series-Library#355]
in https://github.com/thuml/Time-Series-Library/blob/8782a590950b4ee31a0d2d66fc30b761689c3636/exp/exp_imputation.py#L61-L67
the codes are as follows:
```python
                f_dim = -1 if self.args.features == 'MS' else 0
                outputs = outputs[:, :, f_dim:]
                pred = outputs.detach().cpu()
                true = batch_x.detach().cpu()
                mask = mask.detach().cpu()
                # if d_dim==-1, error occurs since the shape not match between pred and mask
                loss = criterion(pred[mask == 0], true[mask == 0])
```

I suggest to modify them so that MS works fine:
```python
                f_dim = -1 if self.args.features == 'MS' else 0
                outputs = outputs[:, :, f_dim:]
                mask = mask[:, :, f_dim:]
                true = true[:, :, f_dim:]
                pred = outputs.detach().cpu()
                true = batch_x.detach().cpu()
                mask = mask.detach().cpu()
                # if d_dim==-1, error occurs since the shape not match between pred and mask
                loss = criterion(pred[mask == 0], true[mask == 0])
```

same issue to loss calculation in train and metric in test.

**Comment by wuhaixu2016** [comment]
Good point! 
For our experiments, we do not include the 'MS' setting, which makes this bug unnoticed. Would you please propose a pull request about this?
